manage_breast_screening/participants/tests/system/test_ethnicity_details_form.py

```diff
@@ -46,6 +46,28 @@ def test_entering_ethnicity_details_for_a_participant(self):
         self.then_i_should_be_back_on_the_appointment()
         self.and_the_ethnicity_is_updated_to("Prefer not to say")
 
+    def test_entering_my_own_ethnicity_details(self):
```

Comment:
Can we reword these a bit? It's the participants enthnicity not the users. I don't mean to be overly pedantic I'm just thinking that we'll be outputting these for assurance and it's a bit confusing as it is at the mo.